### ISSUE TITLE
F# TensorFlow Estimator sample shows Exception in the console output

### ISSUE THREAD

**Issue by prathyusha12345** [dotnet/machinelearning-samples#264]
@kevmal The F# sample TensorFlow Estimator shows exception in the console. Please see the below output when running the F# sample

![image](https://user-images.githubusercontent.com/22335043/53189278-fe1d4b80-35bb-11e9-9f57-f3158c8e07cf.png)

Expected result(what you get with C# sample):
---------------------------------------
![image](https://user-images.githubusercontent.com/22335043/53189829-2fe2e200-35bd-11e9-9e81-f7189227a66c.png)

**Comment by prathyusha12345** [comment]
Fixed in this PR https://github.com/dotnet/machinelearning-samples/pull/272

**Comment by prathyusha12345** [comment]
Reopening as this issue is not fixed yet. Attached another PR wrongly.

**Comment by kevmal** [comment]
@prathyusha12345  Are you still getting an exception after #271?

**Comment by prathyusha12345** [comment]
Now I don't see any exception now. Thanks for fixing it. I am closing the issue.
